libraries/SuplaDevice/supla-common/eh.h

```diff
@@ -18,7 +18,7 @@
 #ifndef EH_H_
 #define EH_H_
 
-#if !defined(ESP8266) && !defined(__AVR__) && !defined(_WIN32)
+#if !defined(ESP8266) && !defined(__AVR__) && !defined(_WIN32) && !defined(ESP32)
```

Comment:
This text is too long.
https://github.com/SUPLA/supla-core/commit/6d0d8e018b5d83df94b544a74dd9c7fff40c0ef2
https://github.com/SUPLA/supla-core/commit/5920f7b9708d793ba65a9a35af677ddd0b6c3392